terraform/modules/redis/variables.tf

```
@@ -0,0 +1,27 @@
+variable "cluster_name" {
+  description = "name of the cluster"
```

This description doesn't add anything (and isn't correctly punctuated to boot). I'd suggest either deleting it or at least something like

```suggestion
  description = "Name of Redis cluster to be created."
```

---

ok